vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/ClearValueIT.java

```diff
@@ -41,6 +41,30 @@ public void valueIsCorrectlyCleared() {
                 ClearValuePage.BUTTON_CLEAR_ID, false);
     }
 
+    @Test
+    public void valueIsCorrectlyClearedWithClearButtonBeforeOpened() {
+        String comboBoxId = ClearValuePage.COMBO_BOX_WITH_CLEAR_BUTTON_ID;
+        ComboBoxElement comboBox = $(ComboBoxElement.class).id(comboBoxId);
+        Assert.assertEquals(String.format(
+                "Unexpected selected item label for combo box with id '%s'",
+                comboBoxId), ClearValuePage.INITIAL_VALUE,
+                comboBox.getSelectedText());
+
+        comboBox.$("vaadin-text-field").get(0)
+                .$("[part~='clear-button']").get(0).click();
+        comboBox.openPopup();
```

Comment:
This shouldn't be needed. Now it doesn't test the scenario which we want: clicking the overlay should set the value to null on server-side immediately without needing to open the overlay.

Reply:
Updated.